src/scripts/farming/Plot.ts

```diff
@@ -2,43 +2,434 @@ class Plot implements Saveable {
         this._isUnlocked = ko.observable(isUnlocked);
-        this._boosted = ko.observable(boosted);
         this._berry = ko.observable(berry);
-        this._timeLeft = ko.observable(timeLeft);
+        this._age = ko.observable(age);
+        this._mulch = ko.observable(mulch);
+        this._mulchTimeLeft = ko.observable(mulchTimeLeft);
+
+        this._auras = [];
+        this._auras[AuraType.Growth] = ko.observable(1);
+        this._auras[AuraType.Harvest] = ko.observable(1);
+        this._auras[AuraType.Mutation] = ko.observable(1);
+        this._auras[AuraType.Replant] = ko.observable(1);
 
         this.formattedTimeLeft = ko.pureComputed(function () {
-            return GameConstants.formatTime(Math.ceil(this.timeLeft) / App.game.oakItems.calculateBonus(OakItems.OakItem.Sprayduck));
+            if (this.berry === BerryType.None) {
+                return '';
+            }
+            for (let i = 0;i < 5;i++) {
+                if (this.age < this.berryData.growthTime[i]) {
+                    const timeLeft = Math.ceil(this.berryData.growthTime[i] - this.age);
+                    const growthMultiplier = App.game.farming.getGrowthMultiplier() * this.getGrowthMultiplier();
+                    return GameConstants.formatTime(timeLeft / growthMultiplier);
+                }
+            }
         }, this);
+
+        this.formattedMulchTimeLeft = ko.pureComputed(function() {
+            if (this.mulch === MulchType.None) {
+                return '';
+            }
+            return GameConstants.formatTime(this.mulchTimeLeft);
+        }, this);
+
+        this.auraGrowth = ko.pureComputed(function() {
+            return this.auras[AuraType.Growth];
+        }, this);
+        this.auraHarvest = ko.pureComputed(function() {
+            return this.auras[AuraType.Harvest];
+        }, this);
+        this.auraMutation = ko.pureComputed(function() {
+            return this.auras[AuraType.Mutation];
+        }, this);
+        this.auraReplant = ko.pureComputed(function() {
+            return this.auras[AuraType.Replant];
+        }, this);
+
+        this.formattedAuras = ko.pureComputed(function() {
+
+            let auraStr = '';
+            if (this.auraGrowth() !== 1) {
+                auraStr += `Growth: ${this.auraGrowth().toFixed(2)}x`;
+            }
+
+            if (this.auraHarvest() !== 1) {
+                if (auraStr) {
+                    auraStr += '<br/>';
+                }
+                auraStr += `Harvest: ${this.auraHarvest().toFixed(2)}x`;
+            }
+
+            if (this.auraMutation() !== 1) {
+                if (auraStr) {
+                    auraStr += '<br/>';
+                }
+                auraStr += `Mutation: ${this.auraMutation().toFixed(2)}x`;
+            }
+
+            if (this.auraReplant() !== 1) {
+                if (auraStr) {
+                    auraStr += '<br/>';
```

Comment:
Maybe just personal preference, but in cases like this, I like to collect all of the strings into an array and then
```js
return strings.join('<br/>')
```
at the end, instead of having to check if we already have stuff at each stage.